static/css/components/merge-form.less

```
@@ -28,6 +28,16 @@ div.merge {
     .master {
       background-color: @light-yellow!important;
```

Let's add `position: sticky` here; then we don't need to deal with another class.

```suggestion
      background-color: @light-yellow!important;
      position: sticky;
      top: 0;
```

---

Resolved.  Also removed references to `sticky` class in `authors.html`.